src/test/java/it/cnr/isti/hpc/wikipedia/article/ArticleTest.java

```diff
@@ -123,6 +123,19 @@ public void testExtractingLinksOtherLang(){
 
 	}
 
+    /*
+    * Test for inline ref parsing.
+    * */
+	@Test
+	public void testReferences() throws IOException {
```

Comment:
Might be good to add more tests, e.g. expectations about that's the text of the reference and stuff like this.  Also it seems that the this article has lots of references, so we might have expectations about the number of references extracted and so on ?